### DISCUSSION TITLE
CSharpMath is now a "must have package" for Xamarin.Forms according to this blog post

### DISCUSSION THREAD

**Discussion by Happypig375** [verybadcat/CSharpMath#268] [Show and tell]
https://devcrux.com/blog/15-nuget-packages-for-xamarin-forms-and-net/
:rocket:

**Comment by mauler2025** [comment]
The capability to render mathematics equations must be inherently implemented as the first class citizen in any frameworks that provide text rendering. Mathematics are everywhere. There should be a universal mathematics equation renderer that can be used not only for mobile applications but also for web based applications.



MathJax that depends on DOM manipulation does not work with Blazor.
Using webview controls in Xamarin.Forms and Flutter is not a good idea.

Hopefully the incoming .NET MAUI will support math equation renderer out of the box. :-)
